llvm/include/llvm/IR/ESIMDVerifier.h

```diff
@@ -0,0 +1,30 @@
+//===--------- ESIMDVerifier.h - ESIMD-specific IR verification -----------===//
```

Comment:
For easier potential upstream, I'd suggest to keep it next to other SYCL/ESIMD-specific passes - llvm\lib\SYCLLowerIR, llvm\include\llvm\SYCLLowerIR.

Reply:
Moved to SYCLLowerIR